SU2_CFD/include/solvers/CFVMFlowSolverBase.hpp

```diff
@@ -2118,6 +2126,8 @@ class CFVMFlowSolverBase : public CSolver {
    */
   inline su2double GetTotal_HeatFluxDiff() const final { return Total_HeatFluxDiff; }
 
+  inline su2double GetTotal_CEquivArea() const final { return Total_CEquivArea; }
```

Comment:
Can you add a brief for this?

Reply:
I have moved the description from original location.